src/main/java/edu/cornell/kfs/fp/service/impl/RecurringDisbursementVoucherDocumentServiceImpl.java

```diff
@@ -409,8 +417,137 @@ public Object call() throws WorkflowException {
             throw new RuntimeException("cancelDVAsSystemUser() Unable to cancel DV: " + dvDocumentNumber, e);
         }
     }
+
+    @Override
+    public boolean autoApproveDisbursementVouchersSpawnedByRecurringDvs() {
+        LOG.info("autoApproveDisbursementVouchersSpawnedByRecurringDvs: Entered.");
+        int approvalCount = 0;
+        int errorCount = 0;
+        List <String> dvDocIdCausingError = new ArrayList<String>();
+        Collection <String> dvDocIds = getRecurringDisbursementVoucherSearchDao().findSavedDvIdsSpawnedByRecurringDvForCurrentAndPastFiscalPeriods(getCurrentFiscalPeriodEndDate());
+        if (ObjectUtils.isNotNull(dvDocIds) && !(dvDocIds.isEmpty())) {
+            for (String dvDocId :  dvDocIds) {
+                LOG.info("autoApproveDisbursementVouchersSpawnedByRecurringDvs: Processing start for DV ID:: " + dvDocId);
+                CuDisbursementVoucherDocument dv = null;
+                try{
+                    dv = getDisbursementVoucher(dvDocId);
+                } catch (WorkflowException e) {
```

Comment:
Logging it once should be fine.  Just as long as we can see that we had a DV document number in the details, but can't be retrieved as a DV from the document service.